### ISSUE TITLE
Support pdb files

### ISSUE THREAD

**Issue by arokem** [nipy/nibabel#313]
This is a somewhat niche file-format for streamlines from dMRI tractography. I have an implementation of reading, writing and converting from/to trk files implemented here: https://github.com/vistalab/pdb_files. 
Would integrating this implementation into nibabel be a desirable thing?

**Comment by matthew-brett** [comment]
How niche is the format?   Who is using it?

**Comment by arokem** [comment]
I am not sure. It is the native format for the Quench viewer, which I believe is still being used among some of my academic siblings and nephews/nieces. But they're not really using Python :-( 

The file format has some neat features. It allows storing not only spatial coordinates, but also associated statistics (e.g. FA along the length of the track, or the mean FA per track).

**Comment by MarcCote** [comment]
I've been really busy lately but once #243 is merged, it would be really easy to add support to it.

**Comment by arokem** [comment]
Sorry to come in late on that one. Is there anything I can do to help that
along? I just took a cursory look and have some superficial comments, but
maybe I should read through the discussion first.

**Comment by MarcCote** [comment]
@arokem I just need to address MrBago's latest comment, I'll have some time to work on #243 in July. In the meantime, I would appreciate any comments on the design/code/documentation (I know I missing some documentation, though).

**Comment by effigies** [comment]
Hi @arokem, looks like this hasn't been touched since #391 got merged. Would this still be a useful thing to merge into nibabel?
